diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigYamlTest.kt

```diff
@@ -26,7 +28,7 @@ inline class RulesConfigYamlTest(private val pathMap: Map<String, String> =
         compareRulesAndConfig("diktat-analysis.yml")
         compareRulesAndConfig("diktat-analysis-huawei.yml")
         val thirdConfig = "${System.getProperty("user.dir")}${File.separator}..${File.separator}diktat-analysis.yml${File.separator}"
-        compareRulesAndConfig(thirdConfig, "parent/diktat-analysis.yml")
+        compareRulesAndConfig(thirdConfig, ".../diktat-analysis.yml")
```

Comment:
```suggestion
        compareRulesAndConfig(thirdConfig, "../diktat-analysis.yml")
```
Firstly, it should be `..`, and secondly, how was it working before?)

Reply:
The second argument is only needed to display the error, because we have two files with the same name, but different paths (diktat-analysis.tml). Since there was no error, this variable was not used.

Reply:
Ok, but still please change three dots to two, it looks weird )

Reply:
Ok, fixed)